### PR TITLE
[DDW-1125] Handle list stake pools API endpoint errors

### PR DESCRIPTION
**Pull request by tomothespian**
This PR introduces handlers for stake pools list API endpoint errors.

## Screenshots

Delegation center -Loading stake pools

Stake pools screen -Loading stake pools

Unknown stake pool delegation

Undelegate unknown stake pool

---

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1576146177369100)

**_Storybook testing:_**
- [ ] Testes storybook story: `Decentralization -> Staking -> Delegation Center - Loading`
- [ ] Testes storybook story: `Decentralization -> Staking -> Pools Index - Loading`
- [ ] Testes storybook story: `Decentralization -> Staking -> Undelegate Confirmation - unknownn stake pool`
- [ ] Testes storybook story: `Decentralization -> Staking -> Delegation Center` _(Second wallet is delegated to unknown stake pool)_

**_In-APP testing:_**
Available commands _(Dev tools -> console)_
1. `daedalus.stores.staking._setFakePoller()` - this command will set api error and force loading until next regular api response
2. `daedalus.stores.staking._setFakePoller(true)` - this command will set api error and force loading forever
3. `daedalus.stores.staking._setFakedStakePools()` - this commant will stop pollers and remove first stake pool from stake pools list

- [ ] Go to "Delegation center" screen and run `daedalus.stores.staking._setFakePoller(true)`. You should see loading stake pools spinner and message
- [ ] Go to "Stake pools" screen and run `daedalus.stores.staking._setFakePoller(true)`. You should see loading stake pools spinner and message
- [ ] Reload app and go to "Delegation center" screen. Delegate one wallet to 1st ranked stake pool. Then enter the command `daedalus.stores.staking._setFakedStakePools()` and you should see that wallet is delegated to the `unknown` stake pool. 
- [ ] Continue to last step and undelegate delegated wallet. On the first dialog step, you should see `unknown` instead stake pool name and ticker.
- [ ] Reload app and go to "Delegation center" screen. Delegate some wallet to 1st ranked stake pool. Then enter the command `daedalus.stores.staking._setFakedStakePools()` and you should see that wallet is delegated to the `unknown` stake pool. Then open delegation wizard again and go to "Choose wallet" step. On regular way, in wallets list, you should see the ticker for wallets that are delegated but now when a wallet is delegating to `unknown` stake pool, the ticker is hidden.
---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

File: storybook/stories/staking/StakePools.stories.js [hunk 1]
```diff
@@ -6,7 +6,10 @@ import { action } from '@storybook/addon-actions';
 import StakePools from '../../../source/renderer/app/components/staking/stake-pools/StakePools';
 import STAKE_POOLS from '../../../source/renderer/app/config/stakingStakePools.dummy.json';
 
-export const StakePoolsStory = (props: { currentTheme: string }) => (
+export const StakePoolsStory = (props: {
```
Comment (DeeJayElly) [inline storybook/stories/staking/StakePools.stories.js]: Just a minor thing.

Maybe extract props at the top, like on other stores.

type Props = {
  currentTheme: string,
  isLoading: boolean,
};

Then just use
export const StakePoolsStory = (props: Props) {

Comment (nikolaglumac) [reply storybook/stories/staking/StakePools.stories.js]: @DeeJayElly may I kindly ask you to implement this change please?
@tomothespian has other work at the moment.

Comment (DeeJayElly) [reply storybook/stories/staking/StakePools.stories.js]: @nikolaglumac @tomothespian done

**Comment by a-rukin** [comment]
image
i feel like it's better to use font-family here

**Comment by daniloprates** [comment]
> i feel like it's better to use font-family here

@a-rukin ✅

**Pull request merged**